### PR TITLE
Removed sdx common clone in dockerfile

### PR DESCRIPTION
**Pull request by emilioward**
This PR fixes the issue of a cryptography error when running docker build in sdx-store 

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by emilioward** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:16.04
+FROM onsdigital/flask-crypto-queue
```
Comment (necrophonic) [inline Dockerfile]: Should only have the single `FROM` - don't need the `ubuntu` one

**Review by necrophonic — CHANGES_REQUESTED** [review]

**Review by JamesGardiner — DISMISSED** [review]

**Comment by gemmai95** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

**Review by dcdarrell9 — APPROVED** [review]
Works for me 👍

**Review by necrophonic — APPROVED** [review]

**Pull request merged**